content/en/opentelemetry/agent/_index.md

```diff
@@ -44,6 +48,16 @@ The Datadog Agent with embedded OpenTelemetry Collector offers:
 
 ## Included components
 
+### Support Levels
+
+Datadog provides different levels of support depending on the type of component:
+
+- **Fully Supported Components**: Datadog-owned components such as the [Datadog Connector][44], [Datadog Exporter][38], and [Infra Attribute Processor][50]. These components are fully maintained by Datadog, receive regular updates, and are prioritized for bug fixes and feature enhancements.
+
+- **Community Supported Components**: Standard OpenTelemetry components included with the Agent by default. Datadog ensures these components are secure, stable, and compatible with the Agent. While Datadog provides assistance with configuration and usage, feature development and enhancements for these components are managed through the OpenTelemetry community.
```

Comment:
"Standard OpenTelemetry components included with Agent" 

Can we rephrase it to below? 
"OpenTelemetry components included with Agent (see [list](https://docs-staging.datadoghq.com/brett.blue/embedded-collector-release/opentelemetry/agent/#opentelemetry-collector-components))" 

1.  remove the word "standard" 
2.  add " (see list)" and add a hyperlink to the "list" to direct users to check what list of components we are referring to. 
- Alternatively, we can directly add the hyperlink to "OpenTelemetry Components"